### PR TITLE
Check jenkins log for SEVERE issues in test

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

**Comment by fwilhe** [comment]
This will make jenkins build fail if an invalid plugin vector is provided. I tested with this `plugins.txt`:

```
git-client:1.0.2
git-server:1.7
git:3.9.3
```

Expected output:

```
...
++ grep --quiet SEVERE
Running from: /usr/share/jenkins/jenkins.war
webroot: EnvVars.masterEnvVars.get("JENKINS_HOME")
Jenkins home directory: /var/jenkins_home found at: EnvVars.masterEnvVars.get("JENKINS_HOME")
$ echo $?
141
```

File: jenkins-master/hooks/test [hunk 1]
```diff
@@ -7,3 +7,6 @@ docker run --name cx-jenkins-master-test --detach --publish 8080:8080 \
 
 # Wait up to 5 minutes for Jenkins to be up and fail if it does not return 200 after this time
 docker exec cx-jenkins-master-test timeout 300 bash -c "until curl --fail http://localhost:8080/api/json; do sleep 5; done"
+
+# Check for 'SEVERE' messages in Jenkins log (usually caused by plugin or configuration issues)
+docker logs cx-jenkins-master-test 2> >(grep --quiet SEVERE)
```
Comment (rkamath3) [inline jenkins-master/hooks/test]: What is the expected behavior here?

Comment (fwilhe) [reply jenkins-master/hooks/test]: Non-zero exit code if the term `SEVERE` is in the log. This happens for example when invalid plugin configuration is present. In consequence, the test will fail if the script has a non-zero exit code.

Comment (rkamath3) [reply jenkins-master/hooks/test]: Do you have a link to a working instance? I couldn't execute the command with the given syntax.

Comment (fwilhe) [reply jenkins-master/hooks/test]: Yes, replace the plugins.txt with the snippet above and you can reproduce the error condition.

**Comment by fwilhe** [comment]
Waiting for a second review @SAP/jenkins-team2

**Pull request merged**